code/game/objects/items/weapons/grenades/chem_grenade.dm

```diff
@@ -421,6 +421,26 @@
 	beakers += B1
 	beakers += B2
 
+/obj/item/weapon/grenade/chem_grenade/bioterrorsmoke
+	name = "Bio terror smoke grenade"
+	desc = "Tiger Cooperative chemical smoke grenade. Causes temporary irration, blindness, confusion, autism, and mutations to carbon based life forms. Contains additional spore toxin"
+	stage = READY
+
+/obj/item/weapon/grenade/chem_grenade/bioterrorsmoke/New()
+	..()
+	var/obj/item/weapon/reagent_containers/glass/beaker/bluespace/B1 = new(src)
+	var/obj/item/weapon/reagent_containers/glass/beaker/bluespace/B2 = new(src)
+
+	B1.reagents.add_reagent("cryptobiolin", 75)
+	B1.reagents.add_reagent("potassium", 50)
+	B1.reagents.add_reagent("mutetoxin", 50)
+	B1.reagents.add_reagent("spore", 75)
+	B1.reagents.add_reagent("itching_powder", 50)
+	B2.reagents.add_reagent("phosphorus", 50)
+	B2.reagents.add_reagent("sugar", 50)
+	B2.reagents.add_reagent("mutagen", 200)

```

Comment:
yeah. even if your in it for like 1 tick, you get confused, so its harder too get away.  this is some sarin-level shit.